### ISSUE TITLE
eslintをかけるべきではないか

### ISSUE THREAD

**Issue by yumetodo** [textlint-ja/textlint-rule-ja-no-abusage#15]
Typescriptで書いている以上、eslintをかけるべきではないか。

**Comment by azu** [comment]
ESLintとTypeScriptの機能が被ることが多いので、自分はあんまり併用してないです。
https://github.com/typescript-eslint/typescript-eslint/ がまだ安定してるとは言えないのもあります。(TSLintもほとんど使わない派でした)

このリポジトリにlint-staged+husky+prettierは入れてなかったので入れておきたいです。

`package.json`にいつも入れる設定。
```
  "prettier": {
    "singleQuote": false,
    "printWidth": 120,
    "tabWidth": 4
  },
```

**Comment by yumetodo** [comment]
> https://github.com/typescript-eslint/typescript-eslint/ がまだ安定してるとは言えないのもあります。

2ヶ月位前にTypeSciprt開発に正式に使われだしてますし、いい加減安定したと言っていいのでは・・・？？

設定項目的に不要というのならそれでもいいと思うのですが。

**Comment by azu** [comment]
https://github.com/typescript-eslint/typescript-eslint/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3Abug+is%3Aissue
ESLintに比べるとパーサとかfalse-positive系のバグがあるきがします。
https://github.com/eslint/eslint/issues?utf8=%E2%9C%93&q=is%3Aopen+label%3Abug+is%3Aissue

**Comment by azu** [comment]
自分はあまり興味がないので、このIssueは閉じます。
必要でしたら、Re-openしてPRを送ってください。
